content/Lab1/30_Deploy_LoadBalancers/index.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: "Deploy Network Loadbalancer"
-weight: 50
+weight: 30
 ---
 
 In this section, we will deploy AWS Network Load Balancer to load balance traffic to the FTDv firewall in AWS using terraform automation. We will first describe and walk through the relevant terraform code snippets and then proceed with the deployment.
 
-[LoadBalancers Setup - Description](../50_Deploy_LoadBalancers/1_LoadBalancers.md)
+[LoadBalancers Setup - Description](../30_Deploy_LoadBalancers/1_LoadBalancers.md)
 
-[LoadBalancers- Deployment](../50_Deploy_LoadBalancers/2_Deployment.md)
\ No newline at end of file
+[LoadBalancers- Deployment](../30_Deploy_LoadBalancers/2_Deployment.md)
```

Comment:
please rename the respective md titles so that it matches with this text e.g. 1_network_resources.md title should be AWS Network Resources - Description